### PR TITLE
Changed two Typos in the german translation

### PR DESCRIPTION
**Pull request by DreiMu**

### CODE REVIEW

**Comment by reloxx13** [comment]
i made a pr with the compiled files, but did not get accepted now.   
https://github.com/DreiMusketiere/moonlight-qt/pull/1

File: app/languages/qml_de.ts [hunk 1]
```diff
@@ -160,7 +160,7 @@
     <message>
         <location filename="../gui/PcView.qml" line="242"/>
         <source>You cannot pair while a previous session is still running on the host PC. Quit any running games or reboot the host PC, then try pairing again.</source>
-        <translation>Solange eine vorherige Session auf dem Host läuft, kann du nicht Pairen. Beende alle laufenden Apps oder state den Host neu. Dann probiere das pairen nochmal.</translation>
+        <translation>Solange eine vorherige Session auf dem Host läuft, kannst du nicht Pairen. Beende alle laufenden Apps oder starte den Host neu. Dann probiere das pairen nochmal.</translation>
```
Comment (reloxx13) [inline app/languages/qml_de.ts:163]: "Pairen" has to be small (verb)

File: app/languages/qml_de.ts [hunk 2]
```diff
@@ -329,27 +329,27 @@
...
         <location filename="../streaming/session.cpp" line="587"/>
         <source>Your settings selection to force software decoding may cause poor streaming performance.</source>
-        <translation>Software decodierung kann zu schlecher streaming Leitung führen.</translation>
+        <translation>Software Dekodierung kann zu schlecher Streaming Leistung führen.</translation>
     </message>
     <message>
         <location filename="../streaming/session.cpp" line="592"/>
         <source>Using unsupported FPS options may cause stuttering or lag.</source>
-        <translation>Nicht unterstütze FPS Optionen kann stottern verursachen.</translation>
+        <translation>Nicht unterstützte FPS Optionenen können Stottern verursachen.</translation>
```
Comment (reloxx13) [inline app/languages/qml_de.ts:342]: "Optionenen" one -en too much

File: app/languages/qml_de.ts [hunk 3]
```diff
@@ -820,7 +820,7 @@
         <location filename="../gui/SettingsView.qml" line="1207"/>
         <location filename="../gui/SettingsView.qml" line="1253"/>
         <source>Automatic (Recommended)</source>
-        <translation>Automatisch (empfohlen)</translation>
+        <translation>Automatisch (Empfohlen)</translation>
```
Comment (reloxx13) [inline app/languages/qml_de.ts:823]: this is not a new sentences, it should be small. but i think big is "ok" in this case.

**Review by reloxx13 — CHANGES_REQUESTED** [review]
Thank you for the corrections.

File: app/languages/qml_de.ts [hunk 4]
```diff
@@ -930,7 +930,7 @@
     <message>
         <location filename="../gui/main.qml" line="271"/>
         <source>Join our community on Discord</source>
-        <translation>Tritt unserer Community auf Discord bei</translation>
+        <translation>Tritt unserer Discord Community bei</translation>
```
Comment (reloxx13) [inline app/languages/qml_de.ts:933]: The community is not only on discord

**Comment by reloxx13** [comment]
I corrected everything from this PR in weblate   
https://hosted.weblate.org/projects/moonlight/moonlight-qt/de/

File: app/languages/qml_de.ts (hunk 1 above)
Comment (SupervisedThinking) [reply app/languages/qml_de.ts:163]: > kannst du nicht pairen

>Dann probiere das Pairen nochmal

**Comment by cgutman** [comment]
@DreiMusketiere can you verify that the changes I've pulled in through Weblate have addressed your concerns?

**Comment by DreiMusketiere** [comment]
I don't see any Problem yet. If I find one, I will inform you.

**Comment by cgutman** [comment]
Closing this. You can use Weblate to adjust translations further.

**Pull request closed without merging**